src/components/ScoreDelta/index.tsx

```diff
@@ -0,0 +1,73 @@
+import React from 'react';
```

Comment:
Content of this file should live in `src/components/ScoreDelta/ScoreDelta.tsx` we are using `index.ts` only for export